johnsonandjohnson/Bodiless-JS
# [DO NOT MERGE]: Chameleon Component Selector

## Important Note
Don't merge this.  It will be reissued against main branch.
Original PR: https://github.com/johnsonandjohnson/Bodiless-JS/pull/1159

## Changes
- Provide an api to allow configuring `withChameleonButton` to uee a component selector form instead of radio buttons.
- Remove dependency on bodiless ui packagtes in @bodiless/components

## Testing Instructions
There is a test case on the /chameleons page (the 2nd example).  Verify that it functions as expected (you can use component selector to choose a variant).
